platform-api/che-core-api-core/src/main/java/org/eclipse/che/api/core/model/project/type/ProjectType.java

```diff
@@ -33,7 +33,7 @@
 
     Attribute getAttribute(String name);
 
-    boolean getCanBeMixin();
+    boolean canBeMixin();
```

Comment:
ProjectType is just interface on server side and it doesn't have DTO implementation.